### PR TITLE
Fix mistake

### PR DESCRIPTION
**Pull request by itsmikej**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "jimdo/prometheus_client_php",
+    "name": "imj/prometheus_client_php",
```
Comment (krukru) [inline composer.json]: Why would you change this?

Comment (itsmikej) [inline composer.json]: I have some custom changes that depend on packagist, and actually I don't want to merge these changes, later I will rollback this commit. Sorry.

Comment (krukru) [inline composer.json]: You could always instruct composer to use your repository, without needing to change the package name. See https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository

Comment (itsmikej) [inline composer.json]: Follow the way you mentioned, I encountered a similar problem like this:  https://www.reddit.com/r/PHP/comments/2ctjb8/is_it_possible_to_use_composerpackagist_with/cjivn7e/ , so I will temporarily maintain tag on other branches.

**Comment by bracki** [comment]
Which mistake are you fixing? Why do you think it is wrong?

**Comment by itsmikej** [comment]
Sorry for that, maybe this can not be called a mistake, I just think this can be reduced once a function call that is not necessary, thanks for the code you provided, it help me a lot.

**Pull request closed without merging**